code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -31,6 +31,7 @@
 /obj/machinery/atmospherics/components/unary/thermomachine/Initialize(mapload)
 	. = ..()
 	initialize_directions = dir
+	SSair.start_processing_machine(src) //initialize thermomachines to SSairs list of tickable machines
```

Comment:
does this ever stop processing?

Reply:
isn't this already enabled in `/obj/machinery/atmospherics`?

Reply:
Yeah. Fixed in d062ccf